### PR TITLE
fix: subspaces CLI commands and Proto files wrong notations

### PR DESCRIPTION
**Pull request by bragaz**
This PR aim to solve some minor issues that I've found inside the `x/subspaces` module.
1. Updates the `groups` CLI commands examples
2. Fixes wrong notations in `subspaces` models proto file
3. Add missing `GetCmdDeleteSubspace` to the `TxCmd`
4. Add missing `GetCmdQueryUserGroup` to the `QueryCmd`
---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/desmos-labs/desmos/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://docs.cosmos.network/v0.44/building-modules/intro.html)
- [ ] included the necessary unit and integration [tests](https://github.com/desmos-labs/desmos/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by RiccardoM — APPROVED** [review]

**Review by RiccardoM — CHANGES_REQUESTED** [review]
We need a changeset entry here

File: go.mod [hunk 1]
```diff
@@ -55,6 +55,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.1 // indirect
 	github.com/desertbit/timer v0.0.0-20180107155436-c41aec40b27f // indirect
+	github.com/desmos-labs/changeset v0.0.0-20220321084648-83441159ab4f // indirect
```
Comment (RiccardoM) [inline go.mod:58]: This shouldn't be here

File: .changeset/entries/9823041f2fc00cbc264d66d2f4f7b02652004a1296489aa64b6b7b586d6329a5.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+type: fix
+module: x/subspaces
+pull_request: 815
+description: fixed wrong proto files notations and added a missing cli command
```
Comment (RiccardoM) [inline .changeset/entries/9823041f2fc00cbc264d66d2f4f7b02652004a1296489aa64b6b7b586d6329a5.yaml:4]:
```suggestion
description: Fixed wrong Proto files notations and missing CLI command
```

**Review by RiccardoM — CHANGES_REQUESTED** [review]

**Comment by bragaz** [comment]
I found that `GetCmdQueryUserGroup()` was missing and added it

File: x/subspaces/client/cli/query.go [hunk 3]
```diff
@@ -165,13 +166,55 @@ func GetCmdQueryUserGroups() *cobra.Command {
 	return cmd
 }
 
+func GetCmdQueryUserGroup() *cobra.Command {
```
Comment (dadamu) [inline x/subspaces/client/cli/query.go:169]: missing doc here.

**Review by dadamu — APPROVED** [review]

**Pull request merged**